index.html

```diff
@@ -1,3 +1,39 @@
+<<<<<<< .mine

```

Comment:
```suggestion
<<<<<<< .mine
```suggestion
<<<<<<< .mine
```
```

Vic, alguma coisa aconteceu aqui, acho que pode ter dado algum problema e ter vindo caracteres especiaris